ccloud/beginner-cloud/start.sh

```diff
@@ -109,16 +109,14 @@ if [[ $status != 0 ]]; then
 fi
 
 echo -e "\n# Produce 10 messages to topic $TOPIC1"
-echo '(for i in `seq 1 10`; do echo "${i}" ; done) | \'
+echo '(for i in `seq 1 10`; do echo "${i}" ; done) | tee | \'
 echo "timeout 10s ccloud kafka topic produce $TOPIC1"
-(for i in `seq 1 10`; do echo "${i}" ; done) | timeout 10s ccloud kafka topic produce $TOPIC1
+(for i in `seq 1 10`; do echo "${i}" ; done) | tee | timeout 10s ccloud kafka topic produce $TOPIC1
```

Comment:
IIUC is the intent to direct output to both the `ccloud` command as well as stdout?  If so, it doesn't work in my env:

```
[13:11:21] ~/git/cp-all-in-one/cp-all-in-one-cloud(DEVX-1719): (for i in `seq 1 10`; do echo "${i}" ; done) | tee | timeout 10s ccloud kafka topic produce t1
Starting Kafka Producer. ^C or ^D to exit
[13:11:35] ~/git/cp-all-in-one/cp-all-in-one-cloud(DEVX-1719): 
```

Any suggestions/clarifications on intent?

Reply:
Sorry, I should have been more verbose. Yes it's to streamline the script. I'll retest.